CONTRIBUTING.md

```diff
@@ -170,66 +170,22 @@ you only see a transient failure once and your change has no reason
-the `tst` button). Clicking on the `Compare` button at the bottom
-right of the page will create a new page containing summaries and
-raw data comparing the selected runs.
-
-- The git repository containing the data about which benchmarks
-to run, on which compilers, at [https://github.com/OCamlPro/ocamlbench-repo](
-https://github.com/OCamlPro/ocamlbench-repo). This needs to be a valid
-opam 2.0 repository, and contains the benchmarks as normal packages
-and the compilers as versions of the package `ocaml-variants`.
-To add a compiler to the list, you must have a publicly accessible
-version of your branch (if you're making a pull request again the
-compiler, you should have a branch on github that was used to make
-the pull request, that you can use for this purpose).
-Then, you should make a pull request against `ocamlbench-repo`
-that adds a repertory in the `packages/ocaml-variants` sub-folder
-which contains a single `opam` file. The contents of the file
-should be inspired from the other files already present, with
-the main points of interest being the `url` field, which should
-point to your branch, the `build` field that should be adapted
-if the features that you want to benchmark depend on configure-time
-options, and the `setenv` field that can be used to pass compiler
-options via the `OCAMLPARAM` environment variable.
-The `trunk+flambda+opt` compiler, for instance, both uses a
-`configure` option and sets the `OCAMLPARAM` variable.
-The folder you add has to be named `ocaml-variants.%VERSION%+%DESCR%`,
-where `%VERSION%` is the version that will be used by opam to
-check compatibility with the opam packages that are needed for the
-benchmarks, and `%DESCR%` should be a short description of the feature
-you're benchmarking (if you're making a pull request against `ocaml`,
-you can use the PR number in the description, e.g. `+gpr0000`).
-Once your pull request is merged, it will likely take a few hours
-until the benchmark server picks up the new definition and again
-up to a few hours before the results are available on the results page.
-
-
 ## Description of the proposed change
 
 ### In the merge request interface
 
 The description of the merge request must contain a precise
 explanation of the proposed change.
 
-Before going in the implementation details, you should include
-a summary of the change, and a high-level description of the design
-of the proposed change, with example use-cases.
+Before going into the implementation details, you should include
+a summary of the change, a justification of why it is beneficial, and
+a high-level description of the design of the proposed change with
+example use cases.
+
+We insist that you should *motivate* the change carefully. Any change
+has a cost; for example, it requires reviewing work and risks
+introducing regressions, it may make the system larger or more
+complex. Explain what are the benefits that compensate for the costs.
```

Comment:
I find this paragraph overly negative, the `insist` reinforced by a `carefully` does give the impression that there some criticism hidden behind the scene.

Would it not be better to focus on the reason why a good motivation is an important of a PR? Something like

> Writing this summary is a good time for introspecting on the design choices and benefits of your PR. Change for the sake of change creates friction with other contributions and may accidentally introduce new bugs. Communicating as clearly as you can the benefits of your PR will reassure and motivate potential reviewers.


or

> Explaining the benefits of your PR plays an important part into motivating other people to reviewing your code. Change for the sake of change creates friction with other contributions and may accidentally introduce new bugs. Taking the time to document your design choice and the benefits brought by your PR will alleviate this fear and motivate people into reviewing your PR.

?

Reply:
I modified the PR following your suggestion. I wasn't fond of "introspecting on" and of "change for the sake of change" (which I find a fairly negative suggestion, more than what I had written before), so I removed parts and the end result is shorter. What do you think?